apps/docs/src/app/platform/component-docs/platform-link/platform-link-examples/platform-link-examples.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'fdp-platform-link-disabled-emphasized-example',
+    templateUrl: './platform-link-disabled-emphasized-example.component.html',
+    encapsulation: ViewEncapsulation.None
+})
+export class PlatformLinkDisabledEmphasizedExampleComponent implements OnInit {
+    ngOnInit() {}
```

Comment:
Do you need `ngOnInit` in the examples? 
*Valid for all examples

Reply:
removed as ngOnInit is unused